examples/wearable/UIComponents/contents/user-inputs-elements/pickers/date-picker.js

```diff
@@ -2,14 +2,29 @@
 (function () {
 	var page = document.getElementById("date-picker-page"),
 		element = page.querySelector(".ui-date-picker"),
-		widget = null;
+		widget = null,
+		setButton = null,
+		SAVED_DATE_KEY = "SAVED-DATE-KEY",
+		savedDateValue = null;
 
 	function init() {
 		widget = tau.widget.DatePicker(element);
+		setButton = page.querySelector(".ui-number-picker-set");
```

Comment:
You can try to add listener on "change" event on element without `captur` flag.

Reply:
Right. Fixed.